test-jest/cartesian/Bar.spec.tsx

```diff
@@ -12,34 +13,34 @@ describe('<Bar />', () => {
     { x: 170, y: 50, width: 20, height: 50, value: 100, label: 'test' },
   ];
 
-  it('Render 5 rectangles in a simple Bar', () => {
+  it(`Render ${data.length} rectangles in a simple Bar`, () => {
     const wrapper = render(
```

Comment:
Nit: Instead of using the wrapper, best use the screen https://stackoverflow.com/questions/61482418/react-testing-library-screen-vs-render-queries

Reply:
```js
it('Will create a background Rectangle with the passed in props', () => {
  const { container } = render(
    <Surface width={500} height={500}>
      <Bar data={composedDataWithBackground} background={{ fill: '#000' }} dataKey="value" />
    </Surface>,
  );

  expect(container.querySelectorAll('.recharts-bar-background-rectangle')).toHaveLength(
    composedDataWithBackground.length,
  );
})
```

I can't find a way to use `screen` for this test. `screen.getByRole` does not allow to select `<svg>` or `<path>` tags, so need to use `wrapper`'s utility. Do you have any recommendations?

Reply:
The way that we are testing with classes its very difficult not to use `container.querySelectorAll`. In order to not @nikolasrieble we'd need to come up with a mechanism for getting things by role, by text, by title, etc. If we had a `title` tag we could query by that easily enough, but since we don't we'll have to rely on this method.

Reply:
+1 - default to screen if it can be used as opposed to destructuring functions like `getAllByText`, etc.